src/System.Net.Primitives/src/System/Net/CookieCollection.cs

```diff
@@ -20,29 +19,20 @@ internal enum Stamp
             SetToMaxUsed = 3,
         }
 
-        internal int _version;
         private List<Cookie> _list = new List<Cookie>();
 
         private DateTime _timeStamp = DateTime.MinValue;
         private bool _hasOtherVersions;
 
-        private readonly bool _isReadOnly;
-
         public CookieCollection()
         {
-            _isReadOnly = true;
-        }
-
-        internal CookieCollection(bool IsReadOnly)
```

Comment:
I looked at both GitHub and the reference source. This is only used by `HttpListenerResponse` in the desktop framework.

http://referencesource.microsoft.com/#System/net/System/Net/cookiecollection.cs,4cea7b17a46824c1,references

Is that going to be ported to .NET Core?